src/Attachment.tsx

```diff
@@ -41,6 +41,16 @@ export const AttachmentView = (props: {
         return searchPattern.test(contentType); 
     }
 
+    const imageWithActionWithOnLoad = (image: {url: string, tap?: Button}, thumbnailUrl?: string, autoPlay?:boolean, loop?: boolean) => {
```

Comment:
Instead of adding two new functions, let's just use the existing ones, conditionally adding the `onClick` parameter if the optional `tap?` field exists in `image`.

Also, minor stylistic nit: when defining objects and types please put a space after `{` and before `}`